src/Components/BoardGame/styles.scss

```diff
@@ -6,10 +10,14 @@
     padding: 10px;
     border: 2px solid #333;
     background-color: #4eefea;
-    .game-board {
+    .gameSetup {
```

Comment:
why camel case here when you have lower letters with `-` anywhere else

Reply:
Yes I noticed that, I changed it back to lower letters to keep consistency